next.config.js

```diff
@@ -1,115 +1,124 @@
+                ]
+            },
+            {
+                source: '/api/lnauth',
+                headers: [
+                    ...corsHeaders
+                ]
+            },
+            {
+                source: '/api/lnurlp/:slug*',
+                headers: [
+                    ...corsHeaders
+                ]
+            }
         ]
-      },
-      {
-        source: '/api/lnauth',
-        headers: [
-          ...corsHeaders
+    },
+    async rewrites() {
+        return [
+            {
+                source: '/faq',
+                destination: '/items/349'
+            },
+            {
+                source: '/story',
+                destination: '/items/1620'
+            },
+            {
+                source: '/privacy',
+                destination: '/items/76894'
+            },
+            {
+                source: '/changes',
+                destination: '/items/78763'
+            },
+            {
+                source: '/guide',
+                destination: '/items/81862'
+            },
+            {
+                source: '/.well-known/lnurlp/:username',
+                destination: '/api/lnurlp/:username'
+            },
+            {
+                source: '/.well-known/nostr.json',
+                destination: '/api/nostr/nip05'
+            },
+            {
+                source: '/~:sub',
+                destination: '/~/:sub'
+            },
+            {
+                source: '/~:sub/:slug*',
+                destination: '/~/:sub/:slug*'
+            }
         ]
-      },
-      {
-        source: '/api/lnurlp/:slug*',
-        headers: [
-          ...corsHeaders
+    },
+    async redirects() {
+        return [
+            {
+                source: '/statistics',
+                destination: '/satistics?inc=invoice,withdrawal',
+                permanent: true
+            }
         ]
-      }
-    ]
-  },
-  async rewrites () {
-    return [
-      {
-        source: '/faq',
-        destination: '/items/349'
-      },
-      {
-        source: '/story',
-        destination: '/items/1620'
-      },
-      {
-        source: '/privacy',
-        destination: '/items/76894'
-      },
-      {
-        source: '/changes',
-        destination: '/items/78763'
-      },
-      {
-        source: '/guide',
-        destination: '/items/81862'
-      },
-      {
-        source: '/.well-known/lnurlp/:username',
-        destination: '/api/lnurlp/:username'
-      },
-      {
-        source: '/.well-known/nostr.json',
-        destination: '/api/nostr/nip05'
-      },
-      {
-        source: '/~:sub',
-        destination: '/~/:sub'
-      },
-      {
-        source: '/~:sub/:slug*',
-        destination: '/~/:sub/:slug*'
-      }
-    ]
-  },
-  async redirects () {
-    return [
-      {
-        source: '/statistics',
-        destination: '/satistics?inc=invoice,withdrawal',
-        permanent: true
-      }
-    ]
-  }
-})
+    }
+}))
```

Comment:
Didn't take a look at all these changes since I assume most are because of changed indentation.

Will take another look when this was undone